dev/breeze/src/airflow_breeze/utils/selective_checks.py

```
@@ -1409,7 +1409,11 @@ def get_job_label(self, event_type: str, branch: str):
 
         response = requests.get(url, headers=headers, params=payload)
         if response.status_code != 200:
-            get_console().print(f"[red]Error while listing workflow runs error: {response.json()}.\n")
+            try:
+                error_msg = response.json()
+            except requests.exceptions.JSONDecodeError:
```

This (and below) should just catch ValueError

---

I understand that using a builtin exception class allows us to avoid a 3rd party import. However, does it not improve readability when we're explicit about the error we're catching (i.e. "not an arbitrary value error but one that's specific to json decoding")?

Also, could you please explain why specifically `ValueError` is preferred in this case? `IOError` is technically suitable too.

Requests exceptions are defined as follows:
```python
class RequestException(IOError):

class InvalidJSONError(RequestException):

class CompatJSONDecodeError(ValueError):  # originally `JSONDecodeError`, imported as `CompatJSONDecodeError`

class JSONDecodeError(InvalidJSONError, CompatJSONDecodeError):
```

---

I changed this to unblock the PR, but would still appreciate your insight re above @uranusjr.